widgets/gui/qt_left_window.py

```diff
@@ -9,6 +9,9 @@
 class LeftWindow(QWidget):
     def __init__(self, parent):
         super(QWidget, self).__init__(parent)
+        self.parent = parent
+        self.parameter_vals = []  # list to hold parameter values. Updated by update_parameters and fetched by
```

Comment:
instead of making a list. for values it can also make sense to have a list for parameter_widgets. In the property you are going to implement you can iterate over the list of widgets and return the parameter values dict(which you can pass to NIDaq constructor right away).

Reply:
I will do that!